experimental/packages/exporter-trace-otlp-http/test/browser/CollectorTraceExporter.test.ts

```diff
@@ -14,21 +14,22 @@
  * limitations under the License.
  */
 
+import * as core from '@opentelemetry/core';
 import { diag, DiagLogger, DiagLogLevel } from '@opentelemetry/api';
 import { ExportResultCode } from '@opentelemetry/core';
 import { ReadableSpan } from '@opentelemetry/sdk-trace-base';
 import * as assert from 'assert';
 import * as sinon from 'sinon';
 import { OTLPTraceExporter } from '../../src/platform/browser/index';
-
+import { nextTick } from 'process';
```

Comment:
This is a browser test and we should prevent using Node.js API in browser codes.